executor/Cargo.toml

```diff
@@ -25,6 +25,7 @@ indicatif = "0.17.7"
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive", "rc"] }
 
 [dev-dependencies]
+powdr-pipeline.workspace = true
```

Comment:
Is this avoidable?

Reply:
Yes, I think I could manually do all steps that need to happen before witgen. I guess the advantage would be that we depend on fewer crates to build the tests (= not the backend crate), but it would a bit cumbersome to write and maintain.

Do you think it's worth it? If we run all tests anyway, the compilation time would be the same, no? In what scenario would it be faster?